pkg/model/resource/webhooks.go

```diff
@@ -33,6 +33,10 @@ type Webhooks struct {
 
 	// Conversion specifies if a conversion webhook is associated to the resource.
 	Conversion bool `json:"conversion,omitempty"`
+
+	// Spoke contains the spoke versions associated with the resource. If this is not
+	// empty, then it means that this resource is the hub
+	Spokes []string `json:"spokes,omitempty"`
```

Comment:
@Adirio , we had spoken about having resource store the hub version. While implementing it, I realized that in that case, we would have to fetch the spoke resource and update it every time. Instead, we could have the resource store the list of spokes. We anyway update the resource while scaffolding, we would just have to add the new spoke version to the list of spokes which the resource holds. One drawback which I thought this implementation could have, is when we allow multiple hubs in the project (we would have to restrict a version to be a spoke to only one resource). Validation in this case may be a little round about, but I suppose that is still a static validation would still be better than updating the resource every time. Does that sound fine?